cylc/flow/cfgspec/suite.py

```diff
@@ -358,27 +358,37 @@
             supports use of the 365 (never a leap year) and 366 (always a leap
             year) calendars.
         ''')
-        Conf('runahead limit', VDR.V_STRING, desc='''
+        Conf('runahead limit', VDR.V_STRING, 'P5', desc='''
             Runahead limiting prevents the fastest tasks in a suite from
             getting too far ahead of the slowest ones, as documented in
             :ref:`RunaheadLimit`.
 
-            This config item specifies a hard limit as a cycle interval
-            between the slowest and fastest tasks. See also
-            :cylc:conf:`[..]max active cycle points` which defines the limit
-            as a number of cyles.
+            This limit on the number of consecutive spawned cycle points is
+            specified by an interval between the least and most recent: either
+            an integer (e.g. ``P3`` -  works for both :term:`integer cycling`
+            and :term:`datetime cycling`), or a time interval (e.g. ``PT12H`` -
+            only works for datetime cycling). Alternatively, if a raw number is
+            given, e.g. ``7``, it will be taken to mean ``PT7H``, though this
+            usage is deprecated.
 
-            Example: ``PT12H`` - for a 12 hour limit under ISO 8601 cycling.
-        ''')
-        Conf('max active cycle points', VDR.V_INTEGER, 3, desc='''
-            Runahead limiting prevents the fastest tasks in a suite from
-            getting too far ahead of the slowest ones, as documented in
-            :ref:`RunaheadLimit`.
+            .. note::
 
-            It allows up to ``N`` (default 3)
-            consecutive cycle points to be active at any time, adjusted up if
-            necessary for any future triggering.
+               The integer limit format is irrespective of the labelling of
+               cycle points. For example, if the runhead limit is ``P3`` and
+               you have a suite *solely* consisting of a task that repeats
+               "every four cycles", it would still spawn three consecutive
+               cycle points at a time (starting with 1, 5 and 9). This is
+               because the suite is functionally equivalent to one where the
+               task repeats every cycle.
+
+            .. note::
+
+               The runahead limit may be automatically raised if this is
+               necessary to allow a future task to be triggererd, preventing
+               the suite from stalling.
         ''')
+        # TODO: make raw number of hours obsolete and then later allow a raw
+        # number to mean the integer limit ?
```

Comment:
Future plans should be recored in issues. Comments get mixed up, forgotten about and become hard to interpret later. (That said the above applies to issues too, just, hopefully, to a slightly lesser extent.)

```suggestion
```